contribs/gmf/apps/desktop_alt/index.html

```diff
@@ -110,9 +110,9 @@
                 <a class="btn close" ng-click="mainCtrl.printPanelActive = false">&times;</a>
               </div>
               <gmf-print
-                gmf-print-map="mainCtrl.map"
+                gmf-print-map="::mainCtrl.map"
```

Comment:
no real need.. `gmf-print` should use one-way binding

Reply:
That's not the same.
'<' stand for "one-WAY" bindings.
`&` (in js) or `::` (in html) stand for "one-TIME" binding.
I think the more important is the one-time binding for the map and with FredJ, we find that it's more practical to use the html notation.

Reply:
I told to use `one-way` because you are passing a map object, so profit is not so big.

As for me it makes sense only for some data that will never change like captions and(or) calculated expressions

https://docs.angularjs.org/guide/expression#reasons-for-using-one-time-binding